src/test/java/teammates/test/cases/datatransfer/FeedbackResponseCommentAttributesTest.java

```diff
@@ -113,4 +113,15 @@ public void testConvertCommentTextToStringForHtml() {
         String commentText = feedbackAttributes.getCommentAsHtmlString();
         assertEquals("hello Images Link: http:&#x2f;&#x2f;test.com&#x2f;test.png ", commentText);
     }
+
+    @Test
+    public void testGetBackUpIdentifier() {
+        FeedbackResponseCommentAttributes feedbackAttributes = FeedbackResponseCommentAttributes
```

Comment:
`commentAttributes` instead of `feedbackAttributes`?

Reply:
Fixed